docs/contributing/code-documentation.txt

```diff
@@ -286,3 +295,58 @@ Type Declaration
     # @option **kwargs [ String | Array<String> ] :items The items(s) as Strings to include.
     # @option **kwargs [ Integer ] :limit An Integer denoting the limit.
     def buy_groceries(**kwargs)
+
+- **Blocks:** Use ``@yield`` to specify when the method yields to a block.
```

Comment:
Can you reference where in the Ruby stdlib this behavior is defined please?

Reply:
`yield` is defined as `rb_yield` in [vm_eval.c](https://github.com/ruby/ruby/blob/master/vm_eval.c#L1357). Here's the (doc in Japanese)[https://docs.ruby-lang.org/ja/latest/function/rb_yield.html]. I can't find the English equivalent.

Reply:
I meant the `@yield` clause. Looking at this again, the yield parameters should either go in `@yieldparam` etc. or they should be in `@yield` but both (or arbitrary choosing) doesn't make sense to me.